connect/usecases/project/update_project.py

```diff
@@ -1,13 +1,11 @@
 from connect.celery import app as celery_app
 from connect.common.models import Project
 from connect.internals.event_driven.producer.rabbitmq_publisher import RabbitmqPublisher
+from django.conf import settings
```

Comment:
adjust order of imports to be consistent with pep8: https://peps.python.org/pep-0008/#imports

1. Standard library imports.
2. Related third party imports.
3. Local application/library specific imports.